README.md

```diff
@@ -63,7 +63,7 @@ This scenario simulates a system that consists of a cubic box of water molecules
 
 First we initialize the scenario:
 ```
-from inductiva.molecules.scenarios.md_water_box import MDWaterBox
+from inductiva.molecules.scenarios import MDWaterBox
```

Comment:
@david451lima @fabiocruz : the pattern wed had before was actually something like:

```python
from inductiva import fluids

scenario = fluids.scenarios.DamBreak(dimensions=(1., 0.3, 0.3))
```

I think that might be a bit neater than importing a specific class (i.e. the scenario name). No?
Should we try to make it consistent to that example we had for the DamBreak?